.gitmodules

```diff
@@ -14,3 +14,6 @@
 	path = external_libraries/portaudio_sc_org
 	url = https://github.com/supercollider/portaudio.git
 	branch = supercollider
+[submodule "external_libraries/yaml-cpp"]
+	path = external_libraries/yaml-cpp
+	url = https://github.com/supercollider/yaml-cpp.git
```

Comment:
Why not use an upstream yaml-cpp tag instead of a personal branch? There's one for yaml-cpp-0.5.3 (we could try against yaml-cpp-0.6.1 as well).

Reply:
@Apteryks - good question. I personally don't have a problem with using upstream, but that was @bagong's choice. We've tended to keep our own forks of repositories for redundancy/reliability.

I haven't been following closely - is there anything new in 0.6.1 that you think we'd benefit from?

Reply:
No special reason to bump to 0.6.1, but if it's free (no issues) and allows to update the yaml-cpp external_libraries submodule to a proper release tag, that'd be a good opportunity.